test/system/src/main/java/io/pravega/test/system/framework/kubernetes/K8sClient.java

```diff
@@ -745,8 +745,10 @@ public void close() {
         public void onFailure(ApiException e, int responseCode, Map<String, List<String>> responseHeaders) {
             if (CONFLICT.getStatusCode() == responseCode || NOT_FOUND.getStatusCode() == responseCode) {
                 log.warn("Exception observed for method {} with response code {}", method, responseCode);
+                log.warn("\tResponseBody: {}", e.getResponseBody());
```

Comment:
These two statements provide the _actual_ error message generated by the call, instead of just a basic `ApiException` thrown and a response code, which I found particularly unhelpful,

Reply:
For example here a call that returns a 404 response:
`
2021-05-26 20:17:47,892 1225 [OkHttp https://10.100.200.1/...] WARN  i.p.t.s.f.kubernetes.K8sClient - ResponseBody: {"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"zookeeperclusters.zookeeper.pravega.io \"zookeeper\" not found","reason":"NotFound","details":{"name":"zookeeper","group":"zookeeper.pravega.io","kind":"zookeeperclusters"},"code":404}
`

Instead of:

`
2021-05-26 04:18:21,676 1084 [OkHttp https://10.100.200.1/...] ERROR i.p.t.s.f.kubernetes.K8sClient - Exception observed for method createCustomObject with response code 400                                                                                                                                                                       io.kubernetes.client.openapi.ApiException:
`